### PR TITLE
docs(Readme): Add documentation about the exposed DOM API available to webapps

### LINKED ISSUE #358: Update README with documentation about Peruse DOM API
Add a new section in Peruse README file to explain that the DOM API exposed is simply the one exposed by safe-app-nodejs with a couple of exceptions which shall be detailed.
This will suffice until we have a better place to publish DOM API documentation.

### PR DESCRIPTION
**Pull request by bochaco**
Resolves #358

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,22 +11,72 @@ For normal SAFE Network browsing, you should download the latest version of the
...
-`NODE_ENV=dev open Peruse.app --args --live`
+```bash
+open Peruse.app --args --live
+```
+
+### SAFE Network API
+
+Peruse exposes a set of APIs in the DOM which webapps can make use to connect to the SAFE Network, as well as fetch and store data on it.
+
+A webapp have direct access to this set of APIs thru the DOM at `window.safe`.
```
Comment (joshuef) [inline README.md]: > have

`has`

> thru

`through`

File: README.md [hunk 2]
```diff
@@ -11,22 +11,72 @@ For normal SAFE Network browsing, you should download the latest version of the
...
+
+The SAFE Network client API exposed by Peruse is a simple wrapper on top of the API provided by the `@maidsafe/safe-node-app` package (with a few minor exceptions explained below), therefore the documentation available for the `safe-node-app` API is valid and t's what a developer creating a webapp for Peruse needs to use as a reference.
```
Comment (joshuef) [inline README.md]: > and t's what a developer creating a webapp for Peruse needs to use as a reference.

`and is the main reference for any developer wanting to create a webapp for The SAFE Network.`

File: README.md [hunk 3]
```diff
@@ -11,22 +11,72 @@ For normal SAFE Network browsing, you should download the latest version of the
...
+
+This API documentation can be found at the following URL: https://docs.maidsafe.net/safe_app_nodejs
+
+As an example, if a webapp is trying to make use of the [initialiseApp](https://docs.maidsafe.net/safe_app_nodejs/#initialiseapp) function, it simply needs to prefix the function name with `window.safe`, i.e. it shall simply call `window.safe.initialiseApp` and provide the parameters as described in that documentation. Note that it's not needed to import/require the `safe-app-node` package with `require('@maidsafe/safe-node-app')` from a webapp.
+
+You will find some example code snippets as well you can also use to learn and try the API.
```
Comment (joshuef) [inline README.md]: On the docs site?

File: README.md [hunk 4]
```diff
@@ -11,22 +11,72 @@ For normal SAFE Network browsing, you should download the latest version of the
...
+
+As mentioned above, there are only a few functions related to the initialisation and app authorisation request process that are exposed by Peruse and which slightly differ from the `safe-app-node` API:
+* There are no [initialisation options](https://docs.maidsafe.net/safe_app_nodejs/#initoptions) supported by the [initialiseApp](https://docs.maidsafe.net/safe_app_nodejs/#initialiseapp) function exposed in the DOM API
+* There are no [initialisation options](https://docs.maidsafe.net/safe_app_nodejs/#initoptions) supported by the [fromAuthUri](https://docs.maidsafe.net/safe_app_nodejs/#fromauthuri) function exposed in the DOM API
+* [openUri](https://docs.maidsafe.net/safe_app_nodejs/#authinterfaceopenuri) from authorisation interface is not available in the DOM API. A webapp shall instead call the `window.safe.authorise` function to send an authorisation request to the Authenticator. E.g.:
```
Comment (joshuef) [inline README.md]: > from authorisation interface is not available in the DOM API.

More simply: `is not available in the DOM API`

**Comment by bochaco** [comment]
Thanks a lot @joshuef , I'll make those corrections shortly. I was also thinking of adding some details about the RDF experimental APIs

**Pull request merged**